doc/bibliography/zz_bibliography.rst

```diff
@@ -9,4 +9,4 @@
    contain citations then those citations aren't handled correctly.
    We therefore put a "zz" at the start of the name of the current file
    to ensure it is processed last.
-.. bibliography:: references.bib
+.. bibliography:: ../bibliography/references.bib
```

Comment:
Add a note saying that this file is included from other locations and the path is relative to those locations, not this one.

Reply:
Done.